### PR TITLE
feat(export): Add possibility to specify input and output delimiters

### PR DESCRIPTION
**Pull request by junajan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -388,6 +388,9 @@ productType,name.en,variantId
     --categoryBy                  Define which identifier should be used to for the categories column - either slug or externalId. If nothing given the named path is used.
     --filterVariantsByAttributes  Query string to filter variants of products
     --filterPrices  Query string to filter prices of variants
+    --templateDelimiter <delimiter> Delimiter used in template | default: ,
+    --cellDelimiter <delimiter>   Delimiter used to sepparate cells in output file | default: ,
```
Comment (PhilippSpo) [inline README.md]: should be: "Delimiter used to separate cells in output file"

Comment (PhilippSpo) [inline README.md]: I'm not sure about the naming of these two options. Will `cellDelimiter` also influence the delimiter setting for the template if the `templateDelimiter` is not set? Do we really need two setting for the delimiter?

File: src/coffee/export.coffee [hunk 2]
```diff
@@ -243,12 +242,14 @@ class Export
     if file then fs.writeFileAsync file, content, opts
     else process.stdout.write content
 
-  _parse: (csvString) ->
-    new Promise (resolve, reject) ->
-      Csv().from.string(csvString)
+  _parse: (csvString) =>
+    new Promise (resolve, reject) =>
+      csvString = _.trim(csvString, @options.templateDelimiter)
```
Comment (PhilippSpo) [inline src/coffee/export.coffee]: why is this needed? Can you explain and add comment?

File: README.md (hunk 1 above)
Comment (junajan) [inline README.md]: it is because templates are now created with "," delimiter which is default but for example excel is expecting ";" - so thats why we have input and output delimiter .. there should be also third one which would specify delimiter of multiple values in one cell but I did not find any way how to specify it in CSV module which we are using

about naming convention: what about "inDelimiter" and "outDelimiter" ?

Comment (junajan) [inline README.md]: updated here c5082df

File: src/coffee/export.coffee (hunk 2 above)
Comment (PhilippSpo) [inline src/coffee/export.coffee]: But if there is an empty cell **on purpose**? Like no value for a specific attribute?

File: README.md (hunk 1 above)
Comment (PhilippSpo) [inline README.md]: Ok. How about `templateDelimiter` and `outputDelimiter`?

File: src/coffee/export.coffee (hunk 2 above)
Comment (junajan) [inline src/coffee/export.coffee]: This `_parse` method is used only for parsing provided CSV template

Comment (PhilippSpo) [inline src/coffee/export.coffee]: 👍  good point 🙈

File: README.md (hunk 1 above)
Comment (junajan) [inline README.md]: sounds good

Comment (junajan) [inline README.md]: updated here 400b0db
